exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py

```diff
@@ -19,26 +19,18 @@
 
 from grpc import ChannelCredentials, Compression
 
+from opentelemetry.exporter.otlp.proto.common.trace_encoder import encode_spans
 from opentelemetry.exporter.otlp.proto.grpc.exporter import (
     OTLPExporterMixin,
     _get_credentials,
-    _translate_key_values,
     environ_to_compression,
-    get_resource_data,
 )
 from opentelemetry.proto.collector.trace.v1.trace_service_pb2 import (
     ExportTraceServiceRequest,
 )
 from opentelemetry.proto.collector.trace.v1.trace_service_pb2_grpc import (
     TraceServiceStub,
 )
-from opentelemetry.proto.common.v1.common_pb2 import InstrumentationScope
-from opentelemetry.proto.trace.v1.trace_pb2 import (
-    ScopeSpans,
-    ResourceSpans,
-)
-from opentelemetry.proto.trace.v1.trace_pb2 import Span as CollectorSpan
-from opentelemetry.proto.trace.v1.trace_pb2 import Status
```

Comment:
I am not sure if we want to remove these. We didn't really think about this when publishing the 1.0 packages. I don't know how this package is being used in the wild. If somebody is importing these transient dependencies it breaks their code and we want to avoid them

Reply:
OK so I'll keep all public members, is there a way to mark them easily for deprecation other than comments? Do I have to keep the public members in `_log`?

Reply:
Also, talking of `_log` should the underscore be added to the new log export package?

Reply:
@jack-burridge-cfh 

> Also, talking of _log should the underscore be added to the new log export package?

Which new package are you referring to?

> OK so I'll keep all public members, is there a way to mark them easily for deprecation other than comments? Do I have to keep the public members in _log?

Probably don't even have to do anything to deprecate them since the symbols themselves are still relevant. I think you can just leave them in there without doing anything (probably have to ignore them for lint purposes since they are not going to be used in our actual code). For `_logs` you can remove them since they are not stable yet.

Reply:
> Which new package are you referring to?

I'm referring to `opentelemetry.exporter.otlp.proto.common.log_encoder`

Reply:
I've run the public symbols checker and noticed it doesn't pick up on the transient dependency removals, should there be a better method of testing this? One that actually understands the package structure, and can use something like `dir` on the module to work out each public member.

Reply:
I'd be willing to add it here as I'm scared I'd be removing something without noticing it

Reply:
@jack-burridge-cfh 

> I'm referring to opentelemetry.exporter.otlp.proto.common.log_encoder

Yes all files/folders related to logging should be prepended with an underscore.

Reply:
> I would leave the public symbols in

I agree its just there currently is no test for public symbols that are transient, as in `tox -e public-symbols-check` doesn't test for those kinds of public symbols sadly

Reply:
regarding the public symbol, yes, the current one doesn't find the transient deps. it has come up in the past to make it more robust https://github.com/open-telemetry/opentelemetry-python/pull/1816#issuecomment-843326648 but we couldn't find any existing tooling and didn't have enough resources to invest in it. so far it is with the reviews we try to catch and make sure it doesn't introduce breaking changes.